openforcefield/tests/test_forcefield.py

```diff
@@ -746,7 +746,7 @@ def test_parameterize_ethanol_missing_torsion(self):
         topology = Topology.from_openmm(pdbfile.topology, unique_molecules=molecules)
         with pytest.raises(UnassignedProperTorsionParameterException,
                            match='- Topology indices [(]5, 0, 1, 6[)]: '
-                                 'names and elements [(] H[)], [(] C[)], [(] C[)], [(] H[)],') \
+                                 'names and elements [(](H\d+)? H[)], [(](C\d+)? C[)], [(](C\d+)? C[)], [(](H\d+)? H[)],') \
```

Comment:
If people are going to rely on these atom numbers in larger workflows, it may be good to test for the exact value we expect instead of the `\d+` regex. So, we'd look to match `H5` `C0` etc here.

Reply:
I don't think we want users to rely on the atom names we generate following any sort of specified pattern. That would be very dangerous, since the order in which different cheminformatics tools might generate atoms from SMARTS strings is totally arbitrary.

Reply:
Sounds good.